packages/scene-composer/src/components/panels/scene-components/data-overlay/DataBindingMapEditor.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useCallback } from 'react';
+import { Box, Button, ExpandableSection, FormField, Input, SpaceBetween } from '@awsui/components-react';
+import { useIntl } from 'react-intl';
+import { isEmpty } from 'lodash';
+
+import { IValueDataBinding, IValueDataBindingProvider } from '../../../../interfaces';
+import { IDataOverlayComponentInternal } from '../../../../store';
+import { ValueDataBindingBuilder } from '../ValueDataBindingBuilder';
+import { generateUUID } from '../../../../utils/mathUtils';
+
+interface IDataBindingMapEditorProps {
+  valueDataBindingProvider: IValueDataBindingProvider | undefined;
+  component: IDataOverlayComponentInternal;
+  onUpdateCallback: (componentPartial: Partial<IDataOverlayComponentInternal>, replace?: boolean | undefined) => void;
+}
+
+export const DataBindingMapEditor: React.FC<IDataBindingMapEditorProps> = ({
+  valueDataBindingProvider,
+  component,
+  onUpdateCallback,
+}) => {
+  const intl = useIntl();
```

Comment:
Use destructuring for all these:

```
const { formatMessage } = useIntl();
```

And get rid of all the namespacing we have to do in the component.